SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -223,6 +224,23 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
     appSecretLabel?.stringValue = (UserDefaults.standard.object(forKey: kMSAppSecret) ?? appCenter.appSecret()) as! String
   }
 
+  @IBAction func clearCrashUserConfirmation(_ sender: Any) {
+    let alert: NSAlert = NSAlert()
+    alert.messageText = "Clear crash user confirmation?"
+    alert.addButton(withTitle: "OK")
+    alert.addButton(withTitle: "Cancel")
+    alert.alertStyle = NSWarningAlertStyle
+    switch(alert.runModal()) {
+    case NSAlertFirstButtonReturn:
+        UserDefaults.standard.removeObject(forKey: kMSUserConfirmationKey)
+        break
+    case NSAlertSecondButtonReturn:
+        break
```

Comment:
This case can be removed

Reply:
Yes, this case can be removed, default case include it .

Reply:
@MatkovIvan Yes, this case can be removed, default case include it .